src/components/Parcels/ParcelDetails/ParcelDetails.jsx

```diff
@@ -34,6 +38,8 @@ const ParcelDetails = (props) => {
         }
     }
 
+    const statuses = [...Object.values(parcelStatusesValues).filter(status => status !== AppConstants.exceptionStatusName)];
```

Comment:
@RamiMustaklem I think this line should be outside of the component function. Now it's being run on every re-render without a reason...

Reply:
Definitely. Done.